app/routes/customer_routes.py

```diff
@@ -0,0 +1,118 @@
+from app import db
+from app.models.customer import Customer
+from .helper_functions import *
```

Comment:
It's generally recommended you don't use `*` except when doing manual testing, the reason being that someone else could add a function with a name that conflicts with something you are using without you realizing.